.github/workflows/Build.yaml

```diff
@@ -96,7 +96,7 @@ jobs:
           cd build
           echo $CMAKE_PREFIX_PATH
           echo "CREATE_PACKAGE: ${CREATE_PACKAGE}"
-          cmake -DCMAKE_BUILD_TYPE=Release -DBLA_VENDOR=${BLA_VENDOR} -DCREATE_PACKAGE=${CREATE_PACKAGE} -DVCPKG_TARGET_TRIPLET=x86-windows -DCMAKE_TOOLCHAIN_FILE=${VCPKG_INSTALLATION_ROOT}/scripts/buildsystems/vcpkg.cmake -G "Visual Studio 16 2019" -A Win32 ..
+          cmake -DCMAKE_BUILD_TYPE=Release -DBLA_VENDOR=${BLA_VENDOR} -DCREATE_PACKAGE=${CREATE_PACKAGE} -DVCPKG_TARGET_TRIPLET=x86-windows -DCMAKE_TOOLCHAIN_FILE=${VCPKG_INSTALLATION_ROOT}/scripts/buildsystems/vcpkg.cmake -G "Visual Studio 17 2022" -A Win32 ..
         shell: bash
```

Comment:
It sure would be nice is this could be a variable that was available in GA. It would be great if you could use $latest or something like that instead of having to hard code the VS version to use.